Snowflake.Data.Tests/UnitTests/SFFileTransferAgentTests.cs

```diff
@@ -487,15 +490,17 @@ public void TestUploadThrowsArgumentExceptionForMissingRootDirectoryWithWildcard
 
             // Set command to upload
             _responseData.command = CommandTypes.UPLOAD.ToString();
+            _responseData.queryId = Guid.NewGuid().ToString();
             _fileTransferAgent = new SFFileTransferAgent(_putQuery,
                 _session,
                 _responseData,
                 _cancellationToken);
 
             // Act
-            Exception ex = Assert.Throws<ArgumentException>(() => _fileTransferAgent.execute());
+            SnowflakeDbException ex = Assert.Throws<SnowflakeDbException>(() => _fileTransferAgent.execute());
 
             // Assert
+            Assert.AreEqual(_responseData.queryId, ex.QueryId);
```

Comment:
Maybe add IO_ERROR_ON_GETPUT_COMMAND check (like in TestDownloadThrowsErrorFileNotFound)?
And maybe rename the test.